javascript/javascript2/week1/averageMedian/average_median.js

```diff
@@ -0,0 +1,68 @@
+const housePrices = [3000000,  3500000,  1300000,  40000000,  100000000,  8000000, 2100000];
+
+// to calculate average
+function getAverage(housePrices){
+
+  let sumHousePrice=0;
+
+  for(let i=0; i<housePrices.length; i++){
+
+      sumHousePrice+=housePrices[i];
+  }
+
+  const averageHousePrice=parseInt(sumHousePrice/housePrices.length);
+
+    return averageHousePrice;
+
+};
+
+
+// to get median -
+
+// median is a number on the center of list of ordered numbers or
+
+// average of the two cetered numbers
+
+function getMedian(housePrices){
+
+    let median=0;
+
+    housePrices.sort();
+
+    if(housePrices.length/2===0){//median  is average of the two ceter numbers
+
+        median=(housePrices[housePrices.length/2 - 1] + housePrices[housePrices.length/2])/2 ;
+
+
+    }else {
+
+        median=housePrices[(housePrices.length +1)/2];
+
+    }
+
+    return median;
+
```

Comment:
Good work. But code to calculate mid value (housePrices.length/2) is repeating every where, you can do it one time and reuse it.

Reply:
Thank you for the feedback. And I made the changes.